css/_add-item.scss

```diff
@@ -0,0 +1,50 @@
+.add-to-do {
+  position: relative;
+  width: 360px;
+  height: 40px;
+  background-color: $white;
+  padding: 10px;
+  border-top: 1px solid $border-shadow;
+
+  i {
+    position: absolute;
+    font-size: 40px;
+    color: $blue;
+  }
+
+  input {
+    position: absolute;
+    left: 50px;
+    height: 35px;
+    width: 310px;
+    background-color: transparent;
+    border: none;
+    font-size: 20px;
+    padding-left: 10px;
+
+    &::-webkit-input-placeholder {
+      /* Chrome/Opera/Safari */
+      color: $blue;
+      font-family: "Titillium Web", sans-serif;
+      font-size: 20px;
+    }
+    &::-moz-placeholder {
+      /* Firefox 19+ */
+      color: $blue;
+      font-family: "Titillium Web", sans-serif;
+      font-size: 20px;
+    }
+    &:-ms-input-placeholder {
+      /* IE 10+ */
+      color: $blue;
+      font-family: "Titillium Web", sans-serif;
+      font-size: 20px;
+    }
+    &:-moz-placeholder {
+      /* Firefox 18- */
+      color: $blue;
+      font-family: "Titillium Web", sans-serif;
+      font-size: 20px;
+    }
```

Comment:
This can be deleted since SCSS creates this automatically for you

Reply:
done